{
  "stats": {
    "suites": 1,
    "tests": 1,
    "passes": 0,
    "pending": 0,
    "failures": 1,
    "start": "2021-08-23T02:00:52.336Z",
    "end": "2021-08-23T02:00:52.341Z",
    "duration": 5
  },
  "tests": [
    {
      "title": "has been correctly cloned to your local environment",
      "fullTitle": "This assignment has been correctly cloned to your local environment",
      "duration": 2,
      "currentRetry": 0,
      "err": {
        "message": "no \".git\" folder was found within \"welcome-completing-assignment\". Use \"git init\" to create one: value: expected './.git' to exist",
        "showDiff": false,
        "actual": "./.git",
        "stack": "AssertionError: no \".git\" folder was found within \"welcome-completing-assignment\". Use \"git init\" to create one: value: expected './.git' to exist\n    at Function.<anonymous> (node_modules/chai-fs/lib/assertions/directory.js:21:53)\n    at Function.ctx.<computed> [as directory] (node_modules/chai/lib/chai/utils/addMethod.js:41:25)\n    at Function.assert.isDirectory (node_modules/chai-fs/lib/assertions/directory.js:34:35)\n    at Context.<anonymous> (test/index-test.js:8:17)\n    at processImmediate (node:internal/timers:464:21)",
        "multiple": [
          {
            "name": "AssertionError",
            "message": "no \".git\" folder was found within \"welcome-completing-assignment\". Use \"git init\" to create one: value: expected './.git' to exist",
            "showDiff": false,
            "actual": "./.git",
            "multiple": "AssertionError: no \".git\" folder was found within \"welcome-completing-assignment\". Use \"git init\" to create one: value: expected './.git' to exist",
            "stack": "AssertionError: no \".git\" folder was found within \"welcome-completing-assignment\". Use \"git init\" to create one: value: expected './.git' to exist\n    at Function.<anonymous> (node_modules/chai-fs/lib/assertions/directory.js:21:53)\n    at Function.ctx.<computed> [as directory] (node_modules/chai/lib/chai/utils/addMethod.js:41:25)\n    at Function.assert.isDirectory (node_modules/chai-fs/lib/assertions/directory.js:34:35)\n    at Context.<anonymous> (test/index-test.js:8:17)\n    at processImmediate (node:internal/timers:464:21)"
          }
        ]
      }
    }
  ],
  "pending": [],
  "failures": [
    {
      "title": "has been correctly cloned to your local environment",
      "fullTitle": "This assignment has been correctly cloned to your local environment",
      "duration": 2,
      "currentRetry": 0,
      "err": {
        "message": "no \".git\" folder was found within \"welcome-completing-assignment\". Use \"git init\" to create one: value: expected './.git' to exist",
        "showDiff": false,
        "actual": "./.git",
        "stack": "AssertionError: no \".git\" folder was found within \"welcome-completing-assignment\". Use \"git init\" to create one: value: expected './.git' to exist\n    at Function.<anonymous> (node_modules/chai-fs/lib/assertions/directory.js:21:53)\n    at Function.ctx.<computed> [as directory] (node_modules/chai/lib/chai/utils/addMethod.js:41:25)\n    at Function.assert.isDirectory (node_modules/chai-fs/lib/assertions/directory.js:34:35)\n    at Context.<anonymous> (test/index-test.js:8:17)\n    at processImmediate (node:internal/timers:464:21)",
        "multiple": [
          {
            "name": "AssertionError",
            "message": "no \".git\" folder was found within \"welcome-completing-assignment\". Use \"git init\" to create one: value: expected './.git' to exist",
            "showDiff": false,
            "actual": "./.git",
            "multiple": "AssertionError: no \".git\" folder was found within \"welcome-completing-assignment\". Use \"git init\" to create one: value: expected './.git' to exist",
            "stack": "AssertionError: no \".git\" folder was found within \"welcome-completing-assignment\". Use \"git init\" to create one: value: expected './.git' to exist\n    at Function.<anonymous> (node_modules/chai-fs/lib/assertions/directory.js:21:53)\n    at Function.ctx.<computed> [as directory] (node_modules/chai/lib/chai/utils/addMethod.js:41:25)\n    at Function.assert.isDirectory (node_modules/chai-fs/lib/assertions/directory.js:34:35)\n    at Context.<anonymous> (test/index-test.js:8:17)\n    at processImmediate (node:internal/timers:464:21)"
          }
        ]
      }
    }
  ],
  "passes": []
}